specs/BadgeNotifications/BadgeNotifications-spec.md

```diff
@@ -0,0 +1,213 @@
+
+The `BadgeNotification` class is used to create badge notifications for apps on platforms such as
+Windows, where badges can be displayed on the app's tile. A badge can show a numeric value or a
+system-provided glyph, providing a visual indicator of some status or count directly on the app's
+icon.
+
+## BadgeNotification constructors
+
+| Name                                      | Description                                                                                                                                                                              |
+| ----------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| BadgeNotification(UInt32)                 | Creates a new `BadgeNotification` object with a numeric badge value. This is used to display a specific number on the app's badge.                                                       |
+| BadgeNotification(BadgeNotificationGlyph) | Creates a new `BadgeNotification` object with a glyph value. This uses a predefined glyph from the `BadgeNotificationGlyph` enum to display a system-provided symbol on the app's badge. |
+
+## BadgeNotification Properties
+
+| Name       | Description                                                                                                                                                                                                  | Type                        |
+| ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------- |
+| Expiration | Gets or sets the time after which the badge notification should no longer be displayed. This allows for the badge notification to be time-sensitive and automatically removed when it is no longer relevant. | Windows.Foundation.DateTime |
+
+## BadgeNotificationManager class
+
+The `BadgeNotificationManager` class is the central point for managing badge notifications for an
+application. It provides the functionality to update or clear the badge displayed on the app's tile.
+This class allows developers to modify the badge's appearance by setting a numeric value or a glyph
+and also to remove the badge when it is no longer needed.
+
+## BadgeNotificationManager Properties
+
+| Name    | Description                                                                                                        | Type                     |
+| ------- | ------------------------------------------------------------------------------------------------------------------ | ------------------------ |
+| Default | Provides a default instance of the `BadgeNotificationManager` for use in updating or clearing badge notifications. | BadgeNotificationManager |
```

Comment:
Sample and API call this "Current", not Default.

Reply:
Corrected the spec